cypress/integration/patch/action-buttons.ts

```diff
@@ -0,0 +1,91 @@
+// / <reference types="Cypress" />
+// / <reference path="../../support/index.d.ts" />
+
+import { mockErrorResponse } from "../../utils/mockErrorResponse";
+
+const patchId = "5e4ff3abe3c3317e352062e4";
+
+describe("Patch Action Buttons", () => {
+  before(() => {
+    cy.login();
+  });
+  beforeEach(() => {
+    cy.preserveCookies();
+    cy.visit(`/patch/${patchId}`);
+  });
+
+  it("Clicking 'Schedule' button shows popconfirm and banner on success", () => {
+    cy.dataCy("schedule-patch").click();
+    cy.get(".ant-btn.ant-btn-primary.ant-btn-sm")
+      .contains("Yes")
+      .click({ force: true });
+    cy.dataCy("banner").should("exist");
+  });
+
+  it("Error scheduling a version shows error banner", () => {
+    cy.dataCy("schedule-patch").click();
+    mockErrorResponse({ errorMessage: "There was an error scheduling tasks" });
+    cy.get(".ant-btn.ant-btn-primary.ant-btn-sm")
```

Comment:
nit. what about selecting these with the data-cy tag instead.
data-cy or some variant using wrapProps should work. 
 https://github.com/ant-design/ant-design/issues/8561#issuecomment-397424511

hesitant on using class names in case we add another element  with a similar class in the future. I like the granularity data-cy allows us to have over selecting elements.

Reply:
the thing about `wrapProps` is that how does it know to pass which data-cy attributes to which children? for example the yes/no buttons on the popconfirm. 

i also tried using the `okButtonProps` prop on the popconfirm but it only accepts specified button props. 

i am fine with the current approach for now.

Reply:
Thats fair. I wish antd supported this more/provided more options for it.